core/src/main/java/org/testcontainers/utility/TestcontainersConfiguration.java

```diff
@@ -65,7 +65,7 @@ public String getAmbassadorContainerImage() {
 
     @Deprecated
     public String getSocatContainerImage() {
-        return (String) properties.getOrDefault("socat.container.image", "alpine/socat:latest");
+        return String.valueOf(properties.getOrDefault("socat.container.image", "alpine/socat:latest"));
```

Comment:
Perhaps:
```suggestion
        return String.valueOf(properties.getOrDefault("socat.container.image", "alpine/socat:latest")).trim();
```

Reply:
Thanks for pointing this out, I should have checked code one more to see if I forgot to trim something :)